apps/hexagon_matmul/Makefile

```diff
@@ -1,23 +0,0 @@
-include ../support/Makefile.inc
```

Comment:
If you put "all:" as the first line of the Makefile, a plain "make" will do the right thing.